### PR TITLE
Fixing bug in brick dashboard creation  when new volume is created

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/monitoring-integration#241

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@rishubhjain @r0h4n @shtripat please review

File: tendrl/monitoring_integration/flows/update_dashboard/alert_utils.py [hunk 1]
```diff
@@ -88,8 +88,16 @@ def add_resource_panel(alert_rows, cluster_id, resource_type, resource_name):
                     pass
             panel["id"] = panel_count
             panel_count = panel_count + 1
+            new_title = resource_name
+            if resource_type == "bricks":
+                new_title = str(
+                    new_title.split("|",1)[1].split(":",1)[0].replace(".","_")
+                )
+                new_title = str(
+                    new_title.split("|",1)[1].split(":",1)[1].replace("/","|")
```
Comment (shtripat) [inline tendrl/monitoring_integration/flows/update_dashboard/alert_utils.py]: Do we need this whole replaces to be done after L#94 done already?

**Comment by GowthamShanmugam** [comment]
@shtripat changes done

**Comment by GowthamShanmugam** [comment]
@r0h4n please review

**Pull request merged**